app/routes/content.js

```diff
@@ -6,6 +6,15 @@ import config from 'ember-get-config';
 import loadAll from 'ember-osf/utils/load-relationship';
 import permissions from 'ember-osf/const/permissions';
 
+// Error handling for API
+const handlers = new Map([
+    // format: ['Message detail', 'page']
+    ['Authentication credentials were not provided.', 'forbidden'], // 401
+    ['You do not have permission to perform this action.', 'forbidden'], // 403
```

Comment:
Suggest having both 401 and 403 errors redirect to Not Found instead of Forbidden. 

This issue was previously raised in this ticket: https://openscience.atlassian.net/browse/EOSF-179.
That ticket said, "Expected behavior for a preprint project made private is that the preprint link will show a "Page not found". 

Forbidden is more accurate, but do we want to tell the user this?